generics/helpers/common_handler_v2.js

```diff
@@ -0,0 +1,755 @@
+const fs = require('fs')
+const { v4: uuidv4 } = require('uuid')
+const { ChartJSNodeCanvas } = require('chartjs-node-canvas')
+const width = 800 //px
+const height = 450 //px
+const chartJSNodeCanvas = new ChartJSNodeCanvas({ width, height })
+const path = require('path')
+const rimraf = require('rimraf')
+const ejs = require('ejs')
+const request = require('request')
+const rp = require('request-promise')
+const filesHelper = require('../../module/files/helper')
+
+exports.unnatiEntityReportPdfGeneration = async function (entityReportData) {
+	console.log(entityReportData, 'line no 11')
+
+	return new Promise(async function (resolve, reject) {
+		console.log('--------yyy---')
```

Comment:
@MallanagoudaB clean up code every where, add relevant comments and function signature. Check all the functions you added